site/docs/pages/index.mdx

```diff
@@ -100,7 +100,7 @@ bun add @coinbase/onchainkit
 :::
 
     </div>
-      <a href="/getting-started" title="View OnchainKit docs" className="font-sans border rounded-xl border-solid border-indigo-600 text-indigo-600 hover:border-indigo-700 dark:border-indigo-400 dark:text-indigo-400 dark:hover:border-indigo-300 px-4 py-3 font-bold mt-4">
+      <a href="/getting-started" title="View OnchainKit docs" className="font-sans border rounded-xl border-solid border-indigo-600 text-indigo-600 hover:bg-gray-200 dark:hover:bg-gray-700 dark:border-indigo-400 dark:text-indigo-400 dark:hover:border-indigo-300 px-4 py-3 font-bold mt-4">
```

Comment:
ok, nit for next PR.
We should create a component for those links, so we can re-use them and avoid those copy and paste of CSS.